Conduct.md

```diff
@@ -1,12 +1,12 @@
-Like the technical community as a whole, the various JS Foundation teams and communities are made up of a mixture of professionals and volunteers from all over the world, working on every aspect of the mission - including mentorship, teaching, and connecting people.
+# TC39 Code of Conduct
 
-Diversity is one of our huge strengths, but it can also lead to communication issues and unhappiness. To that end, we have a few ground rules that we ask people to adhere to. This code applies equally to founders, mentors and those seeking help and guidance.
+Like the technical community as a whole, TC39 is made up of a mixture of professionals and volunteers from all over the world. Diversity is one of our goals, but it can also lead to communication issues and unhappiness. To that end, we have a few ground rules that we ask people to adhere to. This code applies equally to all TC39 delegates and those seeking help and guidance.
 
 This isn't an exhaustive list of things that you can't do. Rather, take it in the spirit in which it's intended - a guide to make it easier to enrich all of us and the technical communities in which we participate.
 
-This Code of Conduct applies to all spaces managed by the JS Foundation and its projects. This includes IRC, the mailing lists, the issue trackers, JS Foundation events, and any other forums created by the project teams which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+This Code of Conduct applies to all spaces managed by TC39. This includes IRC, mailing lists, issue trackers, TC39 events, and any other forums created by TC39 delegates which the community uses for communication. In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
```

Comment:
Enforcement is definitely an open question throughout this doc. I'm going to land this and make a note of that in the README.